python/neuroglancer/pipeline/task_queue.py

```diff
@@ -183,21 +190,43 @@ def cloud_insertion(api):
         else:
             cloud_insertion(self._api)
 
+        return self
+
+    def wait(self):
+        return self.wait_until_queue_empty()
+
     def wait_until_queue_empty(self):
         self._queue.join()
+        return self
 
     def get(self):
         """
-        Gets the named task in a TaskQueue.
+        Gets information about the TaskQueue
         """
-        raise NotImplemented
+        return self._api.taskqueues().get(
+            project=self._project,
+            taskqueue=self._queue_name,
+            getStats=True,
+        ).execute(num_retries=6)
+
+    def get_task(self, tid):
```

Comment:
So far we are hiding this task id from the user inside our `RegisteredTask` , should we make this method take on instance of `RegisteredTask` instead?